CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.6.0-rc.1] - 2020-08-17
```

Comment:
This is invalid, since the link `[29.6.0-rc.1]` doesn't have an associated link value at the bottom of the document. Normally, we'd use `./scripts/update-changelog` to update the changelog, but I'm honestly not sure how the RC version should be reflected in the changelog

Reply:
I see! I think we can still reflect it in the changelog for this special version. Once we're ready to generate a regular version for this, we will use master branch, so this won't appear in the master branch.